pennylane_lightning/lightning_qubit.py

```diff
@@ -660,7 +660,11 @@ def var(self, observable, shot_range=None, bin_size=None):
         return M.var(observable.name, observable_wires)
 
 
```

Comment:
I just realized, I think what we need to do is copy the attribute, otherwise `DefaultQubit` is also affected:

```suggestion
    operations = DefaultQubit.operations.copy()
    operations.remove("Snapshot")

```

Reply:
Yes, that's true. :+1: